types/validator_set_test.go

```diff
@@ -275,7 +275,7 @@ func randPubKey() crypto.PubKey {
 
 func randValidator_() *Validator {
 	val := NewValidator(randPubKey(), cmn.RandInt64())
-	val.Accum = cmn.RandInt64()
+	val.ProposerPriority = cmn.RandInt64() % MaxTotalVotingPower
```

Comment:
I had to change this too (unrelated to the renaming). Without this, `TestValidatorSetBasic` would panic (and fail) on rare occasions. Namely, if `cmn.RandInt64() > MaxTotalVotingPower`.